tests/end_to_end/workflow/private_attr_both.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+pass
+from openfl.experimental.workflow.interface import FLSpec, Aggregator, Collaborator
+from openfl.experimental.workflow.runtime import LocalRuntime
+from openfl.experimental.workflow.placement import aggregator, collaborator
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+class TestFlowPrivateAttributesBoth(FLSpec):
+    """
```

Comment:
Can we mention that in this test, the private attributes are initialized by both methods: callable as well as dictionary.